src/coreclr-debug/activate.ts

```diff
@@ -21,62 +21,84 @@ export function activate(context: vscode.ExtensionContext, reporter: TelemetryRe
     _reporter = reporter;
     _channel = vscode.window.createOutputChannel('coreclr-debug');
     _util = new CoreClrDebugUtil(context.extensionPath, _channel);
-    
+
     if (CoreClrDebugUtil.existsSync(_util.installCompleteFilePath())) {
         console.log('.NET Core Debugger tools already installed');
         return;
     }
-    
-    if (!isDotnetOnPath()) {
-        const getDotNetMessage = "Get .NET CLI tools"; 
+
+    let dotnetVersion: string = '';
+    let osVersion: string = '';
+    let osRID: string = '';
+    _util.spawnChildProcess('dotnet', ['--info'], _util.coreClrDebugDir(), (data: Buffer) => {
+        var lines: string[] = data.toString().replace(/\r/mg, '').split('\n');
+        lines.forEach(line => {
+            let match: RegExpMatchArray;
+            if (match = /^\ Version:\s*([^\s].*)$/.exec(line)) {
+                dotnetVersion = match[1];
+            } else if (match = /^\ OS Version:\s*([^\s].*)$/.exec(line)) {
+                osVersion = match[1];
+            } else if (match = /^\ RID:\s*([\w\-\.]+)$/.exec(line)) {
+                osRID = match[1];
+            }
+        });
+    }).then(() => {
+        let installer = new debugInstall.DebugInstaller(_util);
+        _util.createInstallLog();
+
+        let runtimeId = getPlatformRuntimeId();
+
+        let statusBarMessage = vscode.window.setStatusBarMessage("Downloading and configuring the .NET Core Debugger...");
+
+        let installStage: string = "installBegin";
+        let installError: string = '';
+        let moreErrors: string = '';
+
+        writeInstallBeginFile().then(() => {
+            return installer.install(runtimeId);
+        }).then(() => {
+            installStage = "completeSuccess";
+            statusBarMessage.dispose();
+            vscode.window.setStatusBarMessage('Successfully installed .NET Core Debugger.');
+        })
+            .catch((error: debugInstall.InstallError) => {
```

Comment:
>catch [](start = 13, length = 5)

The indentation of this block seems weird. Shouldn't this be 4 spaces to the left?